packages/addons/addon-depends/dotnet-runtime-depends/aspnet5-runtime/package.mk

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
```

Comment:
2018 ? did we had this package already ?

Reply:
This was already there as **dotnet** - but I have separated it into dotnet and dotnet3 - so **yes**